python/tk_multi_workfiles/file_save_form.py

```diff
@@ -301,9 +319,10 @@ def _generate_path(self, env, name, version, use_next_version, ext, require_path
 
         # first make  sure the environment is complete:
         if not env or not env.context:
-            raise TankError("Please select a work area to save into...")
-        elif not env.work_template:
-            raise TankError("The working directory has not been defined for this work area.  Please select another work area.")
+            raise TankError("Please select a work area to save into.")
+        if not env.are_templates_configured():
```

Comment:
I thought about that. I was trying to strike a balance between having a super technical message and having something that wasn't too technical. I opted for only mentioning the word "template" since by that point a TD will have to go in the yaml, know that the templates are not properly configured and then will look at the 4 values.